src/test/mir-opt/const_allocation/rustc.main.ConstProp.after.mir

```diff
@@ -26,36 +26,63 @@ fn  main() -> () {
     }
 }
 
-alloc0: DefId(0:3 ~ const_allocation[317d]::FOO[0]): 16 bytes, alignment 8
-00 00 00 00 00 00 00 00 03 00 00 00 00 00 00 00 
-└────────(17)─────────┘ 
-
-alloc17: 72 bytes, alignment 8
-00 00 00 00 __ __ __ __ 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 __ __ __ __ 00 00 00 00 00 00 00 00 02 00 00 00 00 00 00 00 01 00 00 00 2a 00 00 00 00 00 00 00 00 00 00 00 03 00 00 00 00 00 00 00 
-                        └─────────(4)─────────┘                                                 └─────────(8)─────────┘                                                 └────────(13)─────────┘ 
+alloc0 {
+    size: 16
+    alignment: 8
+    static const_allocation[317d]::FOO[0]
+    memory: ├──────alloc17+0──────┤ 03 00 00 00 00 00 00 00 
```

Comment:
Yeah that's nice. :D Works well for 64bit at least.
These tests are going to fail on 32bit though, right?

Reply:
:( yea, forgot about that. Good point. We'd already have such failures if the result of a `size_of::<usize>()` call ever ended up in the MIR body

Reply:
`test --target i686-unknown-linux-gnu` worked pretty well when I tried it, if you want to test 32bit.